featurebyte/query_graph/sql/interpreter/preview.py

```diff
@@ -830,17 +855,20 @@ def construct_describe_sql(
             Column to apply date range filtering on
         stats_names: Optional[List[str]]
             List of statistics to compute. If None, compute all supported statistics.
+        columns_batch_size: Optional[int]
+            Maximum number of columns to include in each query. If None, include all columns in a
+            single query.
 
         Returns
         -------
-        Tuple[str, dict[Optional[str], DBVarType], List[str], List[ViewDataColumn]]
+        DescribeQueries
             SQL code, type conversions to apply on result, row indices, columns
         """
         operation_structure = QueryGraph(**self.query_graph.dict()).extract_operation_structure(
             self.query_graph.get_node_by_name(node_name), keep_all_source_columns=True
         )
 
-        sql_tree, type_conversions = self._construct_sample_sql(
+        sample_sql_tree, type_conversions = self._construct_sample_sql(
```

Comment:
Just to check my understanding. For those data warehouse that does not support sample with seed, it means the describe results may come from different sample sets for different column groups. Am I right?

Reply:
That is true. Though the sample result from `_construct_sample_sql` is based on the `RANDOM` function which supports seed in all the platforms that we support now.